CMakeLists.txt

```diff
@@ -346,6 +346,26 @@ message(STATUS "linux probes: ${ENABLE_PROBES_LINUX}")
 message(STATUS "solaris probes: ${ENABLE_PROBES_SOLARIS}")
 message(STATUS " ")
 
+
+message(STATUS "UNIX PROBES (depending of specific libs):")
+message(STATUS "Unix fileextendedattribute probe: ${OPENSCAP_PROBE_UNIX_FILEEXTENDEDATTRIBUTE}")
+message(STATUS "Unix gconf probe: ${OPENSCAP_PROBE_UNIX_GCONF}")
+message(STATUS "Unix process58 probe: ${OPENSCAP_PROBE_UNIX_PROCESS58}")
+message(STATUS " ")
+
+message(STATUS "LINUX PROBES (depending of specific libs):")
+message(STATUS "Linux systemdunitproperty probe (depends of dbus): ${OPENSCAP_PROBE_LINUX_SYSTEMDUNITPROPERTY}")
+message(STATUS "Linux systemdunitdependency probe (depends of dbus): ${OPENSCAP_PROBE_LINUX_SYSTEMDUNITDEPENDENCY}")
+message(STATUS "Linux selinuxsecuritycontext probe (depends of selinux): ${OPENSCAP_PROBE_LINUX_SELINUXSECURITYCONTEXT}")
+message(STATUS "Linux selinuxboolean probe (depends of selinux): ${OPENSCAP_PROBE_LINUX_SELINUXBOOLEAN}")
+message(STATUS "Linux rpmverifypackage probe (depends of rpm): ${OPENSCAP_PROBE_LINUX_RPMVERIFYPACKAGE}")
+message(STATUS "Linux rpmverifyfile probe (depends of rpm): ${OPENSCAP_PROBE_LINUX_RPMVERIFYFILE}")
+message(STATUS "Linux rpmverify probe (depends of rpm): ${OPENSCAP_PROBE_LINUX_RPMVERIFY}")
+message(STATUS "Linux rpminfo probe (depends of rpm): ${OPENSCAP_PROBE_LINUX_RPMINFO}")
+message(STATUS "Linux partition probe (depends of blkid): ${OPENSCAP_PROBE_LINUX_PARTITION}")
+message(STATUS "Linux dpkginfo probe (depends of aptpkg): ${OPENSCAP_PROBE_LINUX_DPKGINFO}")
+message(STATUS " ")
+
```

Comment:
It would be better if all probes that we have are listed here, including independent and Windows.